### ISSUE TITLE
[Idea] One Figma component, multiple dynamic code snippets supported (one for MUI, one for Ant, etc.)

### ISSUE THREAD

**Issue by dusskapark** [figma/code-connect#48]
Hi there. Thank you for your wonderful open source work.

I would like to comment on the opportunity to use this project in a slightly different way. I work in a React environment, but unfortunately nothing is unified at the detailed framework level. 
For example, some older projects are using legacy versions of the Ant design library, while my newer projects want React code based on MUI 5.

Despite this cobweb of issues that are invisible to the user, our user design must have a consistent look. So I'm hoping this CodeConnect will be the answer.

Currently, I'm focusing on creating language-specific snippets like REACT and SWIFT, but...

I hope to be able to add the ability to create different code snippets for different frameworks like MUI4, MUI5, Ant D 4, Ant D 5, etc.

If this feature already exists, I'd love to get a guide. If it's a feature I haven't thought of yet, I'd like to get input from the community and ask for a positive review.

**Comment by ptomas-figma** [comment]
Hi, thanks for your suggestion. We plan to expose a way to label the code snippets so they can be selected via the dropdown in the inspect panel (where now you select React and SwiftUI). Will this be helpful for your use case?

**Comment by dusskapark** [comment]
Yes, of course. I think it's essential to be able to create different code for different version dependencies.
